style.css

```diff
@@ -101,10 +101,107 @@ nav ul {
     line-height: 2em;
 }
 
+/* main */
+
+main {
+    width: 60%;
+}
```

Comment:
This should have been a "red flag". You already set up a width for main in the other PR. Delete this one.

Also, look in `style.css`, you have this rule 3 times in total. Keep just one.